<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainWindow</class>
 <widget class="QWidget" name="mainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>659</width>
    <height>558</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Allenatore</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>641</width>
     <height>541</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="tab1">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <attribute name="title">
     <string>Crea sessione</string>
    </attribute>
    <widget class="QListWidget" name="tab1_listMuscoli">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>231</width>
       <height>491</height>
      </rect>
     </property>
     <property name="currentRow">
      <number>-1</number>
     </property>
    </widget>
    <widget class="QListWidget" name="tab1_listEsercizi">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>10</y>
       <width>381</width>
       <height>221</height>
      </rect>
     </property>
    </widget>
    <widget class="QListWidget" name="tab1_listSessione">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>280</y>
       <width>381</width>
       <height>221</height>
      </rect>
     </property>
    </widget>
    <widget class="QWidget" name="horizontalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>240</y>
       <width>381</width>
       <height>31</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="tab1_layout">
      <item>
       <widget class="QToolButton" name="tbPlus">
        <property name="text">
         <string>+</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="tbMinus">
        <property name="text">
         <string>-</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="tbUp">
        <property name="text">
         <string>Up</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="tbDn">
        <property name="text">
         <string>Dn</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pbOk">
        <property name="text">
         <string>Ok</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pbCarica">
        <property name="text">
         <string>Carica sessione</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="tab2">
    <attribute name="title">
     <string>Eserciziario</string>
    </attribute>
    <widget class="QListWidget" name="tab2_listEsercizi">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>381</width>
       <height>491</height>
      </rect>
     </property>
    </widget>
    <widget class="QListWidget" name="tab2_listMuscoli">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>10</y>
       <width>231</width>
       <height>491</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
